src/commands.c

```diff
@@ -45,12 +42,10 @@ struct _UzblCommands {
     gboolean         search_forward;
     gchar           *search_text;
 
-#ifdef USE_WEBKIT2
 #if WEBKIT_CHECK_VERSION (2, 5, 1)
```

Comment:
Might want to look at bumping our minimum version up to something like 2.6 or so (in Debian stable; unstable has 2.10).